applications/app/views/fragments/audioBody.scala.html

```diff
@@ -63,13 +54,13 @@
                             @fragments.submeta(audio)
                         </div>
                     </div>
-
-                     @if(audio.fields.body.nonEmpty) {
-                         <div class="content__secondary-column content__secondary-column--media content__secondary-column--audio hide-on-childrens-books-site"
-                         aria-hidden="true">
-                             @fragments.commercial.standardAd("right", Seq("dark"), Map("desktop" -> Seq("300,250", "300,274", "fluid")))
-                         </div>
-                     }
+                    @* 
+                    @*  @if(audio.fields.body.nonEmpty) {
+                    @*      <div class="content__secondary-column content__secondary-column--media content__secondary-column--audio hide-on-childrens-books-site"
+                    @*      aria-hidden="true">
+                    @*          @fragments.commercial.standardAd("right", Seq("dark"), Map("desktop" -> Seq("300,250", "300,274", "fluid")))
+                    @*      </div>
+                    @*  } *@
                 </div>
```

Comment:
Do you want to keep this commented out code or can it be deleted?

Reply:
Yeah it has to stay, I'll put it back in in the next iteration

Reply:
Fair enough